docs/index.md

```diff
@@ -14,47 +14,49 @@ In order to integrate with Lead Safe, the health provider will need to make a RE
+| address1        | Text     | <N/A\>                             | Patient home address (street number, street direction, street name, street type)                                  |
+| address2        | Text     | <N/A\>                             | Patient home address (additional information, e.g. apartment number)                                              |
+| city            | Text     | <N/A\>                             | Patient home address city                                                                                         |
+| state           | Text     | <N/A\>                             | Patient home address state                                                                                        |
+| zip             | Text     | <N/A\>                             | Patient home address zip code                                                                                     |
+| date_of_birth   | Date     | yyyy-mm-dd                         | Patient date of birth                                                                                             |
+| gender          | Text     | M/F/U                              | Patient gender                                                                                                    |
+| race            | Text     | Code Values from Centricity        | Standard ONC Race definitions <br> <br> **Code / description** <br> 1002-5 / American Indian or Alaska Native <br> 2028-9 / Asian <br> 2054-5 / Black or African American <br> 2076-8 / Native Hawaiian or Other Pacific Islander <br> 2106-3 / White <br> UNK / Unknown |
+| ethnicity       | Text     | Code Values from Centricity        | Standard ONC Ethnicity definitions <br> <br> **Code / description** <br> 2135-2 / Hispanic or Latino <br> 2186-5 / Non Hispanic or Latino <br> UNK / Unknown |
+| VISIT ARRAY     |          |                                    |                                                                                                                   |
+| visit.visit_id  | Numeric  | 16 Digit GE ID                     | Unique GE ID for specific visit (DocumentID)                                                                      |
+| visit.date      | Date     | yyyy-mm-dd hh:mm:ss.sss-hh:mm      | Date of visit (RFC 3339 complient date)                                                                           |
+| visit.location  | Text     | <N/A\>                             | Text Abbreviation for facility of visit location                                                                  |
+| visit.provider  | Text     | <N/A\>                             | Full provider name, last name, or just NPI                                                                        |
+| LAB ARRAY       |          |                                    |                                                                                                                   |
```

Comment:
After talking with the team today, we'll also have this as an optional item that does not need to be submitted.